docs/config.md

```diff
@@ -33,9 +33,19 @@ The list of images can be separated by commas or by newlines.
   value: img1@sha256:digest1, img2@sha256:digest2
 ```
 
-When processing a `TaskRun`, Chains will parse through the list, then sign and attest each image.
+Chains also suports signing SBOMs. If using the `IMAGE_URL` and `IMAGE_DIGEST` combination, the
+`TaskRun` can also emit the `IMAGE_SBOM_URL` and the `IMAGE_SBOM_FORMAT` results. The first contains
```

Comment:
It can be in a separate PR but we should probably also add support for Object Results, like we have for Artifact Inputs.

```yaml
xxxSBOM:
    url:...
    format:... 
```